### DISCUSSION TITLE
Improve speed of Assistant API

### DISCUSSION THREAD

**Discussion by warranty99** [openai/openai-python#1772] [Q&A]
I recently started checking out the assistants api, i made my own in python and its going swell, i have one issue, and that is the speed, it takes forever, the status is always 'queued', i made it check every 5 seconds if theres a message. if not, print the status, this is literally the text:

`queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued
queued`

I had used the Openai API before to make a different assistant, though i decided to migrate to the Assistants API the other day, is there any way to fix this? im using gpt-turbo-3.5

**Comment by warranty99** [comment]
Nevermind i've found the problem, thanks.

**Reply by mark-brodie** [reply]
Hi,

Would you mind sharing what you learned about improving the assistant API response time? It seems to be a common problem that many people are encountering.

Thanks,
Mark Brodie

**Reply by CharlyJazz** [reply]
This happens for a lot of people
